packages/permissions/src/hooks/use-is-authorized/use-is-authorized.spec.tsx

```diff
@@ -62,9 +62,9 @@ const testRender = ({
   demandedPermissions: TPermissionName[];
   demandedActionRights?: TDemandedActionRight[];
   shouldMatchSomePermissions: boolean;
-  actualPermissions?: TPermissions;
-  actualActionRights?: TActionRights;
-  actualDataFences?: TDataFences;
+  actualPermissions: TPermissions | null;
+  actualActionRights: TActionRights | null;
+  actualDataFences: TDataFences | null;
```

Comment:
Maybe one time we build a Nullable type for this case 🤞

Reply:
https://github.com/Microsoft/TypeScript/issues/19944#issuecomment-344026764

🤷‍♂️